File: Code/Visitor.Java/src/models/Square.java
package models;

/**
 * This class stores the status of a Square object.
 * In this case, I prefer to create the VisitableSquare class.
 */
public class Square {

    private double _side;
    
    public void setSide(double side) {
        this._side = side;
    }
    
    public double getSide() {
		return _side;
	}

	public Square(double side) {
    	this._side = side;
    }
}

File: Code/Visitor.dotNet/Visitor.dotNet/Visitor.dotNet/Models/Square.cs
namespace Visitor.Models
{
    /// <summary>
    /// This class stores the status of a Square object.
    /// In this case, I prefer to create the VisitableSquare class.
    /// </summary>
    internal class Square
    {
        private double _side;

        public double Side
        {
            get { return _side; }
            set { _side = value; }
        }

        public Square(double side)
        {
            _side = side;
        }

    }
}
